src/main/Plugins/ValidationPlugins/Dns/LookupClientProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Net;
+using DnsClient;
+
+namespace PKISharp.WACS.Plugins.ValidationPlugins.Dns
+{
+    public class LookupClientProvider : ILookupClientProvider
+    {
+        private static readonly Lazy<ILookupClient> _defaultLookupClient = new Lazy<ILookupClient>(() => new LookupClient());
```

Comment:
This doesn't have to be a static variable because the DI container at top level is already registers the service as a single instance.

Reply:
i agree that the net effect is the same if you were to remove `static` here. however, it

- signals intent and
- guards against a future change to the container registration

it also doesn't cost anything. since i'm wrapping the explicit consumption in the provider implementation, i think it's important to leave it as-is.

guidance from [DnsClient.NET](http://dnsclient.michaco.net):

> It is important to note, that it is highly recommended to instantiate the lookup client only once per application and to always use the same reference (singleton).

Reply:
Ok, let's keep it in then.